Makefile

```diff
@@ -251,13 +251,23 @@ test-race: TEST_PACKAGES=$(PACKAGES_NOSIMULATION)
 
 $(TEST_TARGETS): run-tests
 
+SUB_MODULES = $(shell find . -type f -name 'go.mod' -printf '%h\n' | sort)
```

Comment:
this is causing some issues on mac osx `find: -printf: unknown primary or operator`

Reply:
Updated. can u test this again?